.github/workflows/coverity-scan.yml

```diff
@@ -0,0 +1,68 @@
+name: Coverity Scan
+on:
+  schedule:
+    - cron: '0 4 * * 1'  ## every Monday morning
+env:
+  COVERITY_SCAN_TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
+  COVERITY_SCAN_EMAIL: github@stefanlau.com
+  COVERITY_PROJECT: ja2-stracciatella/ja2-stracciatella
+  COV_BUILD_DIR: coverage-build
+  COV_BUILD_RESULTS_FILE: analysis-results.tgz
+
+jobs:
+  coverity_scan:
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up environment
+        shell: bash
+        run: ${{ github.workspace }}/.ci/ci-setup.sh 2>&1
+        env:
+          CI_TARGET: linux
+
+      - name: Download and install Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/cxx/linux64 \
+                --post-data "token=${COVERITY_SCAN_TOKEN}&project=${COVERITY_PROJECT}" \
+                -O cov-analysis-linux64.tar.gz
+          mkdir cov-analysis-linux64
+          tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
+          export PATH=$(pwd)/cov-analysis-linux64/bin:$PATH
+
+      - name: Fixed world writable dirs
+        run: |
+          chmod go-w $HOME
+          sudo chmod -R go-w /usr/share
+
```

Comment:
Why is this needed?

Reply:
I just copied it somewhere without thinking too much. I will remove that.